sdks/python/apache_beam/testing/util.py

```diff
@@ -100,7 +125,8 @@ def _empty(actual):
   return _empty
 
 
-def assert_that(actual, matcher, label='assert_that', reify_windows=False):
+def assert_that(actual, matcher, custom_windowing=None,
```

Comment:
Adding an argument in the middle of existing arguments is backwards incompatible (in this case, it  breaks anyone passing label positionally rather than a as a keyword argument). Also, custom_windowing is probably less common then specifying label, and better belongs next to reify_windows (though it seems redundant with that, is the intent to deprecate it?)

Reply:
Filed https://issues.apache.org/jira/browse/BEAM-4447

Reply:
https://github.com/apache/beam/pull/5541 is fixing the order change.

The intent:
The argument was also changed in https://github.com/apache/beam/pull/5384. In its current form it is a flag to control forced rewindowing into global window or not. We could somehow merge with reify_windows perhaps but for now they are doing a different thing.